cram_urdf_projection/cram_urdf_projection/src/setup-urdf.lisp

```diff
@@ -0,0 +1,97 @@
+;;;
```

Comment:
This file is very robot-specific and is a setup thing. Please move this file into urdf_bringup package.